src/static/js/views/faq.html

```diff
@@ -86,14 +92,14 @@ <h4>Frequently Asked Questions</h4>
         How accurate is the package information that Software Discovery Tool provides?
       </div>
       <div class="list-group-item" ng-show="questionState(9)">
-        <p>The package information is regularly updated. It is accurate as on the date of the update. However, the information might become obsolete between updates. To obtain the most current package information, go to the official Linux distribution websites.</p>
+        <p>The package information is regularly updated. It is accurate as on the date of the update. However, the information might become obsolete between updates. To obtain the most current package information, go to the respective official Operating System websites.</p>
```

Comment:
There isn't really a place to go for z/OS open source software (that's why this tool is so useful!) so I wonder how we can reword this. Maybe treat Linux distributions (which are searchable) the way we had it, and "other" operating systems separately? Something like adding: "The list for open source software on other operating systems is maintained by the community in the software-discovery-tool-data repository."